fMRIVolume/scripts/MotionCorrection.sh

```diff
@@ -44,6 +59,25 @@ case "$MotionCorrectionType" in
     ;;
 esac
 
+
+# Run nonlinear registration if needed
+
+if [ "${ReferenceReg}" == "nonlinear" ] ; then
+  verbose_echo " ... computing nonlinear transform to reference"
+  verbose_echo "     ... generating bold average"
+  ${FSLDIR}/bin/fslmaths ${WorkingDirectory}/${OutputfMRIBasename} -Tmean ${WorkingDirectory}/${OutputfMRIBasename}_avg
```

Comment:
Add comment as to why using the mean here, rather than just extracting the first frame and using that.

Reply:
Added a comment.